code/rounds/InProgressRound.cs

```diff
@@ -173,5 +173,25 @@ private static void LoadPostRound(TTTTeam winningTeam)
                 winningTeam.Color
             );
         }
+
+        private bool CheckMinimumPlayers()
```

Comment:
Can we put this in the base class? or does it work as a `static` method somewhere?

Reply:
If you look into every round, the functions are different from another, afaik just reused one time. But yea, we could put it somewhere, just thought it should be handled individually.

Reply:
This could be a static property I think